lib/base/application.cpp

```diff
@@ -286,25 +292,23 @@ void Application::SetArgV(char **argv)
  */
 void Application::RunEventLoop()
 {
-#ifdef HAVE_SYSTEMD
-	sd_notify(0, "READY=1");
-#endif /* HAVE_SYSTEMD */
-
 	double lastLoop = Utility::GetTime();
 
 	while (!m_ShuttingDown) {
 		if (m_RequestRestart) {
 			m_RequestRestart = false;         // we are now handling the request, once is enough
 
-#ifdef HAVE_SYSTEMD
-			sd_notify(0, "RELOADING=1");
-#endif /* HAVE_SYSTEMD */
-
+#ifdef _WIN32
 			// are we already restarting? ignore request if we already are
 			if (!l_Restarting) {
 				l_Restarting = true;
 				m_ReloadProcess = StartReloadProcess();
 			}
+#else /* _WIN32 */
+			Log(LogNotice, "Application") << "Got reload command, forwarding to umbrella process (PID " << m_UmbrellaProcess << ")";
```

Comment:
Please use a newline+tab before the first shift operator. That follows the other log messages and increases readability. Applies to all log messages in this PR.